lib/core/network/repository/auth_repository.dart

```diff
@@ -66,7 +67,15 @@ class AuthRepository {
         publicKey: userAuthData.publicKey.toString(),
       );
 
-      _saveUserData(UserProfileData(accountName: accountName, userName: userName), userAuthData, false);
+      _saveUserData(
+        UserProfileData(
+          accountName: accountName,
+          userName: userName,
+          network: Network.fromString(inviteLinkData.chain),
```

Comment:
@n13  This needs your review. inviteLinkData.chain is a String, and i am mapping this to an ENUM
Pls review the enum and the string values . make sure the map will succeed

Reply:
I see - I guess with the .name we can't do that but still we could make a little mapper that maps these values and defaults to telos if we can't find it.